components/inputText/style.scss

```
@@ -18,6 +18,10 @@
         font-size: 16px;
         padding: 16px;
         box-sizing: border-box;
+		min-width: 200px;
+		max-width: 668px;
+		min-height: 100px;
+		max-height: 300px;
```

Muy copado! No había visto esto, el único tema al ponerle valores absolutos es que cuando pasas a mobile se te rompe porque podés ir más allá, la mejor solución para esto es darle tanto al min-width como al max-width el valor de 100%, entonces siempre toma el valor del contenedor padre.

Para el height está bueno dejarle editar el ancho por si escriben y quieren ver.

```suggestion
    max-width: 100%;
    min-width: 100%;
```

---

Tenes razón! Me la compliqué al pedo, pero la idea se entendió 😁